xml/obs_basic_workflow.xml

```diff
@@ -735,4 +735,12 @@ At revision 1.</screen>
   </sect2>
  </sect1>
 
+ <sect1 xml:id="sec.obs.basicworkflow.manage-group">
+  <title>Manage Group</title>
+  <para>
+    Users with the role Maintainer can add and remove users from its group. They
```

Comment:
Users with Maintainer rights  can add users to their group and remove users from it.